atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -650,13 +660,158 @@ def set_build_image(self):
             raise RuntimeError("Build object is malformed, failed to fetch buildroot image")
 
         if build_kind == 'DockerImage':
-            self.config_kwargs['build_image'] = build_name
+            return build_name
         else:
             raise RuntimeError("Build kind isn't 'DockerImage' but %s" % build_kind)
 
+    def set_build_image(self, platforms):
+        """
+        Overrides build_image for worker, to be same as in orchestrator build
+        """
+        orchestrator_plat = 'x86_64'
```

Comment:
Let's say `platform` instead of `plat` to reduce eye friction when reading the code.

Reply:
ok